pkgs/development/compilers/dmd/default.nix

```diff
@@ -84,6 +84,13 @@ stdenv.mkDerivation rec {
   postPatch =
   ''
     patchShebangs .
+
+    find | fgrep cdvecfill
```

Comment:
Did you mean to leave this line in here?

Reply:
Oh, good catch! It's a leftover if me being lazy to find out the full paths of deleted files in this phase. Dropped `find | ...` call.